### PR TITLE
More perf

### PR DESCRIPTION
**Pull request by s-cork**
```txt
200 runs avg: 38.64904773ms
     fastest: 32.15404200000012ms 
       first: 116.85512500000002ms
```

`tokenize()` in this branch is approx twice as fast as master.

It also makes us 35% faster than skulpt.

### CODE REVIEW

File: src/tokenize/token.ts [hunk 1]
```diff
@@ -79,52 +79,52 @@ export const ERRORTOKEN = tokens.ERRORTOKEN;
 export const COMMENT = tokens.COMMENT;
 export const NL = tokens.NL;
 
-export const EXACT_TOKEN_TYPES: { [token: string]: tokens } = {
```
Comment (albertjan) [inline src/tokenize/token.ts:82]: so are `Map`'s faster than object access?

File: src/tokenize/tokenize.ts [hunk 2]
```diff
@@ -53,8 +53,7 @@ function rstrip(input: string, what: string): string {
 }
 
 const Whitespace = "[ \\f\\t]*";
-const Comment_ = "#[^\\r\\n]*";
-const _Ignore = Whitespace + any("\\\\\\r?\\n" + Whitespace) + maybe(Comment_);
```
Comment (albertjan) [inline src/tokenize/tokenize.ts:57]: what's going on here? 😄

File: src/tokenize/token.ts (hunk 1 above)
Comment (s-cork) [reply src/tokenize/token.ts:82]: They seemed to be... Or at least that's what the profiler indicated.

File: src/tokenize/tokenize.ts (hunk 2 above)
Comment (s-cork) [reply src/tokenize/tokenize.ts:57]: Just totally unused - exist in tokenize.py - but they're also unused there...

File: src/tokenize/token.ts (hunk 1 above)
Comment (albertjan) [reply src/tokenize/token.ts:82]: maybe because it doesn't have to walk the prototype. I remember you doing something with that in skulpt too.

Comment (LeszekSwirski) [reply src/tokenize/token.ts:82]: if you want to avoid prototype walks you could try `__proto__: null`

**Pull request merged**